Added build instructions

KBbitsP opened the pull request:
Hi @reshmabidikar

KBbitsP committed:
Added build instructions

reshmabidikar reviewed (commented):
Could you also add a link to the latest version on Maven central? See an example of this in the [Stripe plugin](https://github.com/killbill/killbill-stripe-plugin#killbill-stripe-plugin):
<img width="592" alt="image" src="https://user-images.githubusercontent.com/85998496/223018692-00746aae-a97a-457d-b82f-2a09a5c7180e.png">
Secondly, this readme could do with a bit of cleanup. Some specific thoughts:

- Is the [code organization section](https://github.com/killbill/killbill-catalog-plugin-test/#code-organization) really required? If not, feel free to delete
- Maybe have a separate **configuration** section that explains the different configuration options?
- Clean up the [use cases](https://github.com/killbill/killbill-catalog-plugin-test/#use-cases) section as required
- Add a new **Testing** section that clearly explains how this plugin can be tested

KBbitsP committed:
Updated Readme

1) Removed Code organization Section.
2) Added some more details to Use cases.
3) Clubbed and specified instructions clearly for the configuration section.
4) Added testing instructions.

reshmabidikar reviewed (approved):
Looks good!
